apiserver/facades/client/controller/controller.go

```diff
@@ -544,40 +491,43 @@ func (c *ControllerAPI) HostedModelConfigs(ctx context.Context) (params.HostedMo
 		return result, errors.Trace(err)
 	}
 
-	modelUUIDs, err := c.state.AllModelUUIDs()
+	modelIDs, err := c.modelService.ListModelIDs(ctx)
```

Comment:
If this call was combined with the one above to return a slice of `coremodel.Model` the call below wouldn't be needed to pull the info again.